pkg/logger/common/common.go

```diff
@@ -92,7 +93,7 @@ func simpleAccessLogFilter(req *restful.Request, resp *restful.Response, chain *
 	chain.ProcessFilter(req, resp)
 
 	duration := time.Since(start)
-	logrus.Infof(commonLogFormat,
+	fmt.Println(fmt.Sprintf(commonLogFormat,
```

Comment:
is it fine if the logs are always printed? we cannot set the log level in standard print function
However, if you have performance concern by using logrus, wdyt if we use another log library like [Uber Zap](https://pkg.go.dev/go.uber.org/zap)?

Reply:
for now it's fine, logrus.Infof always be printed even if we set level as `debug`/`error`/`fatal`

we could explore Uber Zap too, 
could we put it in separate initiative like "replacing logrus from justice",
since we need to handle the configuration in service side (e.g. setting the LOG_LEVEL), or maybe we can create standard library for it.

Reply:
I dont think so, Info level has lower level than Error / Fatal, it shouldn't be printed if the log level is Error / Fatal

yeah, you're right we can separate the initiative for log library 
however, I think it's better to use logrus for now since it more controllable than standard print

Reply:
for now, I prefer to use logrus instead of creating custom logger since we can utilize the logrus log level from the service

Reply:
for now let's use the option 1 above.

this is the current log format if we're using the default `logrus.Infof`:
`time="2021-09-03T07:46:04.373Z" level=info msg="time=2021-09-03T07:46:04.373Z log_type=access method=GET path=\"/healthz\" status=200 duration=0 length=176 source_ip= user_agent=\"kube-probe/1.16\" referer=\"\" trace_id=e8e77ce78c2341528c4036584fea0c38 namespace= user_id= client_id= request_content_type=\"\" request_body=AB[]AB response_content_type=\"application/json\" response_body=AB[{\"name\":\"justice-group-service\",\"healthy\":true,\"dependencies\":[{\"name\":\"MongoDB\",\"url\":\"mongodb.mongodb:27017\",\"healthy\":true}],\"others\":[]}]AB"`

while what we want to achieve is something like this:
`time=2021-09-03T07:46:04.373Z log_type=access method=GET path="/healthz" status=200 duration=0 length=176 source_ip= user_agent="kube-probe/1.16" referer="" trace_id=e8e77ce78c2341528c4036584fea0c38 namespace= user_id= client_id= request_content_type="" request_body=AB[]AB response_content_type="application/json" response_body=AB[{"name":"justice-group-service","healthy":true,"dependencies":[{"name":"MongoDB","url":"mongodb.mongodb:27017","healthy":true}],"others":[]}]AB`

seems the only way is to create new logrus object with CustomFormatter